dynamiq/memory/backends/qdrant.py

```diff
@@ -31,36 +31,91 @@ class Qdrant(MemoryBackend):
     on_disk: bool = Field(default=False)
     create_if_not_exist: bool = Field(default=True)
     recreate_index: bool = Field(default=False)
-    vector_store: QdrantVectorStore | None = None
+    vector_store: QdrantVectorStore | dict | None = None
```

Comment:
We shouldn't handle some custom dict on init as it could direct to other problems. Please leave possible vector_store params as it was before - `QdrantVectorStore | None`